mono/mini/mini-llvm.c

```diff
@@ -10998,122 +10997,6 @@ mono_llvm_init (void)
 
 /* LLVM JIT support */
 
-static unsigned char*
-alloc_cb (LLVMValueRef function, int size)
-{
-	MonoCompile *cfg;
-
-	cfg = (MonoCompile*)mono_native_tls_get_value (current_cfg_tls_id);
-
-	if (cfg) {
-		// FIXME: dynamic
-		return (unsigned char*)mono_domain_code_reserve (cfg->domain, size);
-	} else {
-		return (unsigned char*)mono_domain_code_reserve (mono_domain_get (), size);
-	}
-}
-
-static void
-emitted_cb (LLVMValueRef function, void *start, void *end)
-{
-	MonoCompile *cfg;
-
-	cfg = (MonoCompile*)mono_native_tls_get_value (current_cfg_tls_id);
-	g_assert (cfg);
-	cfg->code_len = (guint8*)end - (guint8*)start;
-}
-
-static void
-exception_cb (void *data)
-{
-	MonoCompile *cfg;
```

Comment:
If this is removed, not sure where we added the unwind info for llvm compiled code to the runtime unwind tables. This was used by the JIT code for older llvm versions, maybe newer versions were not ported to use it.

Reply:
I've got to sleep, but I'll take a look at this in the morning.

Reply:
This is done here: https://github.com/mono/mono/blob/2146f1f41842af3cf3e4f18ce7f01c0a00b6eba3/mono/mini/mini-llvm.c#L11117-L11204